### DISCUSSION TITLE
Gcal to TickTick Question

### DISCUSSION THREAD

**Discussion by singlellama** [vibalcam/ticktick-gcalendar-py#3] [General]
If I make changes to an event (change date/time) in Google Cal, does this script change the date/time in TickTick as well? 

Is this replicating todoist's version of 2 way sync but with TickTick

**Comment by vibalcam** [comment]
Yes, if you make changes to an event (date/time/name...) in Google Calendar, the script also updates the corresponding event in TickTick. However, take into account that only a single calendar in Google Calendar is synced (you choose which one in the configuration file).

I am not familiar with todoist's 2 way sync. Can you elaborate?
